.travis.yml

```diff
@@ -0,0 +1,10 @@
+sudo: false
```

Comment:
Interesting. But why use both? Currently, CircleCI is used for tests, docker build, Github release uploads, etc - what's the benefit of using both?